packages/neo-one-cli/src/cmd/deploy.ts

```diff
@@ -6,7 +6,7 @@ import { deploy } from '../deploy';
 export const command = 'deploy';
 export const describe = 'Deploys the project using the migration file.';
 export const builder = (yargsBuilder: typeof yargs) =>
-  yargsBuilder.string('network').describe('network', 'Network to run the migration on.').default('network', 'test');
```

Comment:
this should still be 'test'; see https://github.com/neo-one-suite/neo-one/blob/1fe420d7a03f1d32e87a4c56a6871d22a2e2d240/packages/neo-one-cli-common-node/src/networks.ts#L26

Reply:
Ahh. Got it.